README.md

```diff
@@ -43,7 +45,7 @@ To build binary: ```mvn package```
 - which will produce  ```tool/target/datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar```
 
 To use the tool: ```java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>``` 
-- Its useful to create an alias like ```alias dc-import='java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>'```, so you can invoke the tool as ```dc-import```
+- It's useful to create an alias like ```alias dc-import='java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>'```, so you can invoke the tool as ```dc-import```
```

Comment:
May be we can move this to "Using the tool" section?

Reply:
done